src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.cs

```diff
@@ -27,24 +27,73 @@ public sealed partial class JsonDocument : IDisposable
         private byte[] _extraRentedBytes;
         private (int, string) _lastIndexAndString = (-1, null);
 
+        /// <summary>
+        ///   Whether or not this JsonDocument is detached from memory provided during a call to Parse.
```

Comment:
or `ReadFrom`?

Reply:
ReadFrom is always detached :smile:, but right now that API doesn't exist and is still subject to change, so it feels weird to reference it here in case this one wins.